packages/Modal/src/index.tsx

```diff
@@ -55,13 +59,12 @@ const Modal: React.SFC<IModalProps> = ({
     modalHeaderClassNames,
     modalContentClassNames,
     modalFooterClassNames
-  ]: string[] = sections.map(
-    (alertType: string): string =>
-      classNames(`modal__${alertType}`, {
-        [`modal__${alertType}--alert`]: alert,
-        [`modal__${alertType}--simple`]: simple,
-        [`modal__${alertType}--upload`]: upload
-      })
+  ]: string[] = sections.map((alertType: string): string =>
+    classNames(`modal__${alertType}`, {
+      [`modal__${alertType}--alert`]: alert,
+      [`modal__${alertType}--simple`]: simple,
+      [`modal__${alertType}--upload`]: upload
+    })
   );
 
   return (
```

Comment:
This should be controlled by conditional rendering, not by an internal flag.

Reply:
Perfect, a ticket has been added to improve the modal